q2_types/tree/_type.py

```diff
@@ -22,8 +26,32 @@
 
 plugin.register_semantic_types(Phylogeny, Rooted, Unrooted, Hierarchy)
 
-plugin.register_semantic_type_to_format(Phylogeny[Rooted | Unrooted],
-                                        artifact_format=NewickDirectoryFormat)
+plugin.register_artifact_class(
+    Phylogeny[Rooted], NewickDirectoryFormat,
+    "A phylogenetic tree containing a defined root.")
+
+## Phylogeny[Unrooted] import usage example
+def phylogeny_unrooted_usage(use):
+    def factory():
+        from q2_types.tree import NewickFormat
+
+        tree = TreeNode.read(StringIO(
+            '(SEQUENCE1:0.000000003,SEQUENCE2:0.000000003);'))
```

Comment:
Should the root be trifurcating in this particular example, since that's the common-ish convention?

Reply:
(Or we can switch it to `Phylogeny[Rooted]`)

Reply:
Thanks for catching that! I updated to Phylogeny[Rooted] and set this as ready for review.